std/range/package.d

```diff
@@ -7130,6 +7130,17 @@ sgi.com/tech/stl/binary_search.html, binary_search).
         }
         return false;
     }
+
+// groupBy
+/**
+Returns a range of subranges of elements that are equivalent according to the
+sorting relation.
+ */
+    auto groupBy()()
```

Comment:
I do not get this name. If we call the function `chunkBy`, why call the method `groupBy`? If we call the function `chunkBy`, the proper name for the method would be `chunk`.